pkg/AnewLibrary/Data/atom-manifest.js

```diff
@@ -7,20 +7,19 @@
 const category = 'Data';
```

Comment:
general comment: having FileNames alongside files_names is confusing... i try guessing the rule (like AtomFiles and framework_files) but cannot figure it..